core/src/main/java/feast/core/model/Job.java

```diff
@@ -59,17 +61,9 @@
   @JoinColumn(name = "store_name")
   private Store store;
 
-  // FeatureSets populated by the job
-  @ManyToMany(cascade = CascadeType.ALL)
-  @JoinTable(
-      name = "jobs_feature_sets",
-      joinColumns = @JoinColumn(name = "job_id"),
-      inverseJoinColumns = @JoinColumn(name = "feature_sets_id"),
-      indexes = {
-        @Index(name = "idx_jobs_feature_sets_job_id", columnList = "job_id"),
-        @Index(name = "idx_jobs_feature_sets_feature_sets_id", columnList = "feature_sets_id")
-      })
-  private List<FeatureSet> featureSets;
+  // FeatureSets populated by the job via intermediate FeatureSetJobStatus model
+  @OneToMany(mappedBy = "job", cascade = CascadeType.ALL)
+  private Set<FeatureSetJobStatus> featureSets = new HashSet<>();
```

Comment:
Shouldn't this be `featureSetJobStatus`?

Reply:
Don't we call models in plural?